fish_prompt.fish

```diff
@@ -322,7 +322,7 @@ function __bobthefish_prompt_git -a current_dir -d 'Display the actual git state
   set -l new ''
   set -l show_untracked (git config --bool bash.showUntrackedFiles)
   if [ "$theme_display_git_untracked" != 'no' -a "$show_untracked" != 'false' ]
-    set new (command git ls-files --other --exclude-standard)
+    set new (command git status --porcelain)
```

Comment:
this doesn't actually show when there are new files, though, it shows whenever there are any modifications at all :-/

i haven't seen significant slowdowns using this. i'm curious whether adding `--directory` would speed things up for you?